backend/internal/handlers/classrooms/assignments/assignments.go

```diff
@@ -53,6 +53,28 @@ func (s *AssignmentService) getAssignment() fiber.Handler {
 	}
 }
 
+// Returns the template of an assignment.
+func (s *AssignmentService) getAssignmentTemplate() fiber.Handler {
+	return func(c *fiber.Ctx) error {
+		assignmentID, err := strconv.ParseInt(c.Params("assignment_id"), 10, 64)
+		if err != nil {
+			return errs.BadRequest(err)
+		}
+
+		assignment, err := s.store.GetAssignmentByID(c.Context(), assignmentID)
```

Comment:
Is this something that can be simplifed to a simple join between the assignment and assignment template tables? Instead of two seperate db calls